uber/site_sections/jobs.py

```diff
@@ -54,8 +55,11 @@ def index(self, session, location=None, message=''):
             'location':  location,
             'setup':     [j for j in jobs if j.type == c.SETUP],
             'teardown':  [j for j in jobs if j.type == c.TEARDOWN],
+            'normal': [j for j in jobs if j.type != c.SETUP and j.type != c.TEARDOWN],
             'checklist': location and session.checklist_status('creating_shifts', location),
-            'times':     [(t, t + timedelta(hours=1), by_start[t]) for i, t in enumerate(times)]
+            'times':     [(t, t + timedelta(hours=1), by_start[t]) for i, t in enumerate(times)],
+            'jobs': jobs,
+            'c': c
```

Comment:
`c` is a global object that we make available to our templates in the rendering process. You don't need to pass it as a variable here.